Remove `element` parameter from Button component (#6383)
alphagov/govuk-frontend: CHANGELOG.md

This change was introduced in [pull request #6366: Update Dart Sass tests minimum version to 1.79.0](https://github.com/alphagov/govuk-frontend/pull/6366)

#### Stop using the `element` parameter of the Button component

> The `element` parameter of the Button component Nunjucks macro was deprecated in [GOV.UK Frontend 5.1.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.1.0) and has been removed.

Reviewer: ```suggestion
The `element` parameter of the Button component Nunjucks macro was deprecated in [GOV.UK Frontend 5.1.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.1.0), and we've now removed it.
```

> The component is now output as a link if the `href` parameter is set, otherwise it's a `button` element.

Reviewer: ```suggestion
The component is now output as a link if the `href` parameter is set. Otherwise, it's a `button` element.
```
